nixpkgs-overlays/hack-get/default.nix

```diff
@@ -7,13 +7,21 @@ self:
 
   # Retrieve source that is controlled by the hack-* scripts; it may be either a stub or a checked-out git repo
   hackGet = p:
-    let filterArgs = x: removeAttrs x [ "branch" ];
-    in if builtins.pathExists (p + "/git.json") then (
+    let
+      filterArgs = x: removeAttrs x [ "branch" ];
```

Comment:
This is tangental, but we should probably use a whitelist instead of a blacklist for this. It would improve compat with more recent obelisk thunks at least.

Reply:
That sounds like a good idea. I think filterAttrs will do this for us.